scripts/postinstall.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd /var/praekelt/goddard-captive-portal
+npm install
+
+mkdir -p /var/goddard/apps
+ln -s /var/praekelt/goddard-captive-portal /var/goddard/apps/captiveportal-redesign
```

Comment:
So we need to add a check because the build will possibly fail as this can generate an error exit code for all installs but the first one.